Gruntfile.js

```diff
@@ -500,7 +506,8 @@ module.exports = function(grunt) {
     'copy:libphonenumber',
     'browserify:dist',
     'replace:hardcodeappsettings',
-    'ngtemplates'
+    'ngtemplates',
+    'exec:optimizeJs',
```

Comment:
I think it'd be better to just call this when releasing, so maybe add it as part of the `minify` task?